site/blog/airship-update-march-2020.md

```diff
@@ -38,12 +38,38 @@ component. The first six Airship 2.0 Blog posts are already available:
 
 <br>
 
-## **GITHUB ISSUE PROGRESS**
+## **AIRSHIP 2.0 PROGRESS**
+
+Airship is a loosely coupled, interoperable open source tools that declaratively automate cloud provisioning. It is
+designed to make your cloud deployments simple, repeatable, and resilient.
+
+The primary motivation for Airship 2.0 is the continued evolution of the control plane, and by aligning with maturing
+CNCF projects we can improve Airship by making 2.0:
+- More capable
+- More secure
+- More resilient
+- Easier to operate
+
+To do this, we are moving purpose-built service logic developed in Airship 1.0 into native objects living in Kubernetes,
```

Comment:
I think this sentence is grammatically incorrect. Needs to be:

"To do this, we are moving purpose-built service logic developed in Airship 1.0 into native objects living in Kubernetes, building a new client [*airshipctl*](https://opendev.org/airship/airshipctl), and a new UI
[*airshipui*](https://opendev.org/airship/airshipui) to interact with those objects."

Reply:
done.